avaframe/in3Utils/generateTopo.py

```diff
@@ -0,0 +1,556 @@
+"""
+
+  Create generic/idealised topographies - based on run_dem_creator.m JT Fischer (2011)
+
+  This file is part of Avaframe.
+
+"""
+
+
+# load modules
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+from scipy.stats import norm
```

Comment:
scipy needs to be added to the requirements.txt file

Reply:
scipy added in the requirements.txt file